### DISCUSSION TITLE
connecting the CMT2300A.

### DISCUSSION THREAD

**Discussion by PeterLustig33** [tbnobody/OpenDTU#1971] [General]
I have a problem connecting the CMT2300A.

Everything is nicely described here: [https://www.opendtu.solar/hardware/cmt2300a/#description](url)
OpenDTU, but for a different version of the module.

![2024-05-05 10_24_18-CMT2300A-Modul – OpenDTU-Dokumentation](https://github.com/tbnobody/OpenDTU/assets/48348196/fe5a1033-8f4a-40a7-8e18-d7f8f706dfd9)

I have this module:

![71QgK0bnRqL _AC_SL1500_](https://github.com/tbnobody/OpenDTU/assets/48348196/fbb02461-8198-429a-ace0-0e1795e48378)

I can't find the GPIO2 pin. Can someone tell me which pin I need to use?

**Comment by ghost** [comment]
Hi,

wenn ich nicht irre, werden GPIO2 und GPIO3 nicht zwingend benötigt. Sind beide laut der OpenDTU Documentation IRQ Pins.
Der CMT kann 3-Draht Halbduplex bei der SPI-Kommunikation.
![Screenshot 2024-05-06 174453](https://github.com/tbnobody/OpenDTU/assets/155244148/de8dfbb7-0a1b-4719-8b2c-5e6a148068ea)

Ansonsten bleibt an deinem CMT ja nur der Pin NIQR als GPIO2 übrig.

Hier mal ein Beispiel einer PinMapping.json, wo beide Funkmodule aktiviert sind.
    {
        "name": "NRF24 + CMT2300A",
        "nrf24": {
            "miso": 19,
            "mosi": 23,
            "clk": 18,
            "irq": 16,
            "en": 4,
            "cs": 5
        },
        "cmt": {
            "clk": 12,
            "sdio": 14,
            "cs": 27,
            "fcs": 26,
            "gpio2": -1,
            "gpio3": -1
        }
    }

**Reply by ghost** [reply]
-1 bedeutet wohl: nicht Vorhanden/verwendet.
